condor_watch_q.py

```diff
@@ -37,7 +37,14 @@
 
 
 def parse_args():
-    parser = argparse.ArgumentParser(
+    class CustomParser(argparse.ArgumentParser):
```

Comment:
I don't think there's any particular need to have this class defined inside `parse_args`, so let's move it outside.